invenio_communities/assets/semantic-ui/js/invenio_communities/community/frontpage.js

```diff
@@ -181,12 +25,14 @@ if (userCommunitiesContainer) {
     userCommunitiesContainer
   );
 }
-ReactDOM.render(
-  <CommunitiesCardGroup
-    fetchDataUrl="/api/communities?q=&sort=newest&page=1&size=5"
-    emptyMessage="There are no featured communities."
-    defaultLogo="/static/images/square-placeholder.png"
-  />,
-  featuredCommunitiesContainer
-);
-export default CommunitiesCardGroup;
+
+if (newCommunitiesContainer) {
+  ReactDOM.render(
+    <CommunitiesCardGroup
+      fetchDataUrl="/api/communities?q=&sort=newest&page=1&size=5"
+      emptyMessage="There are no featured communities."
```

Comment:
Do we have to change also the text, if `featured` was wrong and `new` is the correct one?

Reply:
Good catch. I've adjusted the text, thanks.